packages/webchat/src/components/messages/Message.tsx

```diff
@@ -173,11 +41,33 @@ class Message extends Component<MessageProps> {
 
     const type = this.props.type || (this.props.payload && this.props.payload.type)
     const wrappedType = this.props.payload && this.props.payload.wrapped && this.props.payload.wrapped.type
-    const renderer = ((this as any)[`render_${type}`] || this.render_unsupported).bind(this)
     const wrappedClass = `bpw-bubble-${wrappedType}`
     const isEmulator = this.props.store!.config.isEmulator
 
-    const rendered = renderer()
+    const rendered = (
+      <ReactMessageRenderer
+        key={this.props.messageId}
+        content={this.props.payload}
+        config={{
+          messageId: this.props.messageId!,
+          noMessageBubble: this.props.noBubble!,
+          isLastOfGroup: this.props.isLastOfGroup!,
+          isLastGroup: this.props.isLastGroup!,
+          isBotMessage: this.props.isBotMessage!,
+          sentOn: this.props.sentOn!,
+          onSendData: this.props.onSendData!,
+          onFileUpload: this.props.onFileUpload!,
+          store: this.props.store,
+          onAudioEnded: this.props.onAudioEnded,
+          shouldPlay: this.props.shouldPlay,
+          intl: this.props.store!.intl,
+          escapeHTML: true,
+          isInEmulator: false,
```

Comment:
I don't think this should be a constant! What will happen once we use this version of the webchat as the emulator?

Reply:
Rigth there's an options passed in the props for that. Changed it